app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -72,6 +73,9 @@ class FocusApplication : LocaleAwareApplication() {
 
             TelemetryWrapper.init(this@FocusApplication)
             AdjustHelper.setupAdjustIfNeeded(this@FocusApplication)
+            launch {
+                TLDExtractor.init(this@FocusApplication)
+            }
```

Comment:
Afaik `launch()` without a coroutine context will run on the current context - which would be the main thread here -> Blocking main thread for reading files -> ANR.